### ISSUE TITLE
the plugin produces a warning in the logs whenever Mojolicious starts

### ISSUE THREAD

**Issue by akarelas** [augensalat/mojolicious-plugin-accesslog#9]
```
==> api.err.log <==
[Sat Nov 21 22:41:50 2015] [warn] Mojolicious::Plugin::AccessLog::VERSION = 0.010
```

I think this is useless, and shouldn't be there since I didn't do anything wrong to get warned about.

**Comment by akarelas** [comment]
And it's pretty annoying when using Morbo (which restarts often)

**Comment by jberger** [comment]
👍  it is getting on the nerves of our ops staff. Even if it is useful, which I'm not convinced about, it certainly isn't warn level.

**Comment by christopherraa** [comment]
@jberger @akarelas I have a (fork)[https://github.com/christopherraa/mojolicious-plugin-accesslog] of this module in which this is fixed, as well as the addition of a placeholder (`%x`) that extracts the outermost client in the X-Forwarded-For chain, if that is of interest to you.

**Comment by Grinnz** [comment]
Agreed with above comments

**Comment by karenetheridge** [comment]
This looks like an easy fix - any chance it can be merged?

**Comment by augensalat** [comment]
Fixed with [release 0.010001](https://github.com/augensalat/mojolicious-plugin-accesslog/releases/tag/0.010001).

**Comment by karenetheridge** [comment]
thanks very much!
